File: EcoJardimPro - BackEnd/EcoJardimProject/EcoJardimProject.Domain/Entities/Servico.cs
using System.ComponentModel.DataAnnotations;
using EcoJardimProject.Domain.Base;

namespace EcoJardimProject.Domain.Entities;

public class Servico: IEntity
{
    public long Id { get; set; }
    public string Nome { get; set; }
    public string Descricao { get; set; }
    public decimal Valor { get; set; }
    public long StatusId { get; set; }
    public Status? Status { get; set; }
    public long OrcamentoId { get; set; }
    public Orcamento? Orcamento { get; set; }
    public string? DataInicio { get; set; }
    public string? DataFinalizacao { get; set; }
}

File: EcoJardimPro - FronEnd/ecojardimprofrontend/src/models/servico/Servico.ts
import Orcamento from "../orcamento/Orcamento"
import Status from "../status/Status"

export default class Servico{
    public id: number
    public nome: string
    public descricao: string
    public valor: number
    public statusId:number
    public status?: Status
    public orcamentoId: number
    public orcamento?: Orcamento
    public dataInicio?:string
    public dataFinalizacao?:string
    constructor() {
        this.id = 0
        this.nome = ''
        this.descricao = ''
        this.valor = 0
        this.statusId = 0
        this.status = new Status()
        this.orcamentoId = 0
        this.orcamento = new Orcamento()
        this.dataInicio = '0001-01-01T00:00:00'
        this.dataFinalizacao = '0001-01-01T00:00:00'
    }
}